IconManip.py

```diff
@@ -98,6 +98,11 @@ def rgbToRGB5a1(rgbValues):
 
 # Patch overworld icons
 def patch_overworld_icon(rom, color, address, fileName = None):
+    if color is None:
+        original = rom.original.read_bytes(address, 0x800)
+        rom.write_bytes(address, original)
+        return
+
     rgbBytes = rgb5a1ToRGB(rom.read_bytes(address, 0x800))
```

Comment:
Not really related, but I just noticed this should probably use the original data, because this process is likely to reduce the quality of the texture during each consecutive patch.

Reply:
Good point, I was just focused on fixing the bug but that's an easy change.